nimble-storage/nimble-csp-api-swagger-v2.yaml

```diff
@@ -790,6 +790,16 @@ definitions:
   ContainerReplicationPartner:
     type: object
     properties:
+      cgfSyncStatus:
+        type: string
+        enum:
+        - N/A
+        - 'No'
+        - 'Yes'
+      volumeCollectionList:
```

Comment:
Should we wait to add this until a use case for it is identified?

Reply:
I had it marked as hidden, not sure why it still spits out.  I will remove it for time being as you suggested
```    @ApiModelProperty(hidden = true)
```